.github/workflows/ci-develop.yml

```diff
@@ -0,0 +1,30 @@
+name: Dokazovi-fe CI on Develop
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - name: Copy all sources
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install, build, test, lint
+        run: |
+          npm install
+          npm build
+          npm test
+          npm run lint
+        env:
+          CI: true
```

Comment:
might be vulnerable, since CI=true treats lint warnings as errors

Reply:
fixed